pyproject.toml

```diff
@@ -37,7 +36,15 @@ documentation = "https://k4black.github.io/fastapi-jwt/"
 
 
 [project.optional-dependencies]
+authlib = [
+    "Authlib >=1.3.0"
+]
+python_jose = [
+    "python-jose[cryptography] >=3.3.0"
+]
```

Comment:
useful for people to be able to do:
pip install fastapi_jwt[authlib]

or 

pip install fastapi_jwt[python_jose]